lib/storage.js

```diff
@@ -343,10 +343,10 @@ zcaps.get = async ({controller, id, referenceId, explain = false} = {}) => {
     return cursor.explain('executionStats');
   }
 
-  const record = collection.findOne(query, {projection});
+  const record = await collection.findOne(query, {projection});
   if(!record) {
     throw new BedrockError(
-      'Authorization capability not found.',
+      'ZCap capability not found.',
```

Comment:
This should either be `ZCap not found.' or 'Authorization capability not found.' not 'ZCap capability' (authorization capability capability :) ). I think we should just leave it as is.

```suggestion
      'Authorization capability not found.',
```